### ISSUE TITLE
Install Failure

### ISSUE THREAD

**Issue by Reichenbachian** [jupyterlab/jupyterlab-google-drive#111]
On attempting to install using ```jupyter labextension install @jupyterlab/google-drive```

I get the response 

```'NoneType' object is not iterable```.

I'm not sure how to go about debugging this. Any ideas?

**Comment by ian-r-rose** [comment]
That sounds like an error on the Python side of JupyterLab. What version of JupyterLab are you using?

**Comment by blink1073** [comment]
That looks like a 0.30 error.  @Reichenbachian, can you please try again with `--debug` to get the full traceback?

**Comment by Reichenbachian** [comment]
I just pip installed jupyterlab today. Here's the end of the traceback. 
```
  File "/home/localhost/Desktop/Projects/Working/Peabody-Notecards/venv/local/lib/python2.7/site-packages/jupyterlab/commands.py", line 1016, in _run
    proc = Process(cmd, **kwargs)

  File "/home/localhost/Desktop/Projects/Working/Peabody-Notecards/venv/local/lib/python2.7/site-packages/jupyterlab/process.py", line 69, in __init__
    self.logger.info('> ' + list2cmdline(cmd))

  File "/home/localhost/Desktop/Projects/Working/Peabody-Notecards/venv/local/lib/python2.7/site-packages/jupyterlab/process.py", line 32, in list2cmdline
    return ' '.join(map(pipes.quote, cmd_list))

  File "/usr/lib/python2.7/pipes.py", line 269, in quote
    for c in file:

TypeError: 'NoneType' object is not iterable

'NoneType' object is not iterable

```

**Comment by Reichenbachian** [comment]
Turns out I don't have npm installed, so the line `ret = self._run([which('npm'), 'pack', source], cwd=tempdir)` has a none value in the iteration in the file `python2.7/site-packages/jupyterlab/commands.py`. Should I branch and push a fix?

**Comment by ian-r-rose** [comment]
As of JupyterLab 0.30 we are distributing a version of yarn with the package, so people do not need to have `npm` installed (though they still need node). The new workflow is to use `jlpm` in place of `npm`.

**Comment by blink1073** [comment]
I think the right fix is to expand this check to handle `npm` as well: https://github.com/jupyterlab/jupyterlab/blob/master/jupyterlab/jlpmapp.py#L57

PRs welcome!

**Comment by ian-r-rose** [comment]
Ah, my mistake. PR away!

**Comment by ian-r-rose** [comment]
Fixed by jupyterlab/jupyterlab#3317
